### PR TITLE
Manage esdoc internal  link to pass through react-router

### PR DESCRIPTION
**Pull request by oterral**
# How to

Manage esdoc internal  link to pass through react-router

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [ ] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

**Comment by oterral** [comment]
omg it doesn't seem to be the laste version

**Comment by oterral** [comment]
@friedjoff  fixed

**Comment by oterral** [comment]
@friedjoff  you can retest

File: esdoc/template/layout.html [hunk 1]
```diff
@@ -20,9 +20,9 @@
   <div class="search-box active">
   <span>
     <!--Geops <img src="./image/search.png"/> -->
```
Comment (friedjoff) [inline esdoc/template/layout.html:22]: I guess we can remove the commented code.

Comment (oterral) [reply esdoc/template/layout.html:22]: I wanted to keep track of what was there before.

**Pull request merged**